targets/qtpy-rp2040-boot-stage2.S

```diff
@@ -0,0 +1,17 @@
+// Adafruit QT Py RP2040 Stage 2 Bootloader
+
+//
+// This file defines the parameters specific to the flash-chip found
+// on the Adafruit QT Py RP2040.  The generic implementation is in
+// rp2040-boot-stage2.S
+//
+
+#define BOARD_PICO_FLASH_SPI_CLKDIV   4
```

Comment:
Maybe CLKDIV works with 2, can you confirm?
If there is a problem with the CLKDIV configuration, the #2960 problem will occur.

https://github.com/raspberrypi/pico-sdk/blob/master/src/boards/include/boards/adafruit_qtpy_rp2040.h#L83

Reply:
Changed to 2, as it is working on 2